web-ui/src/components/reviews/periods/ReviewPeriodCard.css

```diff
@@ -0,0 +1,22 @@
+.review-period-card {
```

Comment:
The UI that renders each review period in `ReviewPeriods.jsx` was factored out to create the new component `ReviewPeriodCard`. This is the CSS for the new component.